aten/src/ATen/test/vulkan_api_test.cpp

```diff
@@ -2670,7 +2670,7 @@ TEST_F(VulkanAPITest, cat_dim1_samefeature_success) {
   ASSERT_TRUE(check);
 }
 
-TEST_F(VulkanAPITest, cat_dim1_difffeature_success) {
+TEST_F(VulkanAPITest, DISABLED_cat_dim1_difffeature_success) {
```

Comment:
why disable this test?

Reply:
is flaky like 10% (T125856190)

let's fix the test later then

Reply:
Sounds good. LGTM!